libmscore/style.cpp

```diff
@@ -872,7 +872,7 @@ static const StyleType styleTypes[] {
       { Sid::frameFontStyle,                "frameFontStyle",               int(FontStyle::Normal) },
       { Sid::frameColor,                    "frameColor",                   QColor(0, 0, 0, 255) },
       { Sid::frameAlign,                    "frameAlign",                   QVariant::fromValue(Align::LEFT) },
-      { Sid::frameOffset,                   "frameOffset",                  QPointF() },
+      { Sid::frameOffset,                   "frameOffset",                  QPointF(0.0,5.0) },
```

Comment:
Don't forget to add whitespace after the comma.

Reply:
yes sir

Reply:
and yes sir , I have signed the CLA.

Reply:
frameOffset too? Why?

Reply:
Sir there seems to be some error. I haven’t added any changes to frameoffset. I just made changes to the footeroffset. I will take care of everything as soon as I get grab of my computer.